char type
uint64 timestamp
float32 px
float32 py
float32 vx
float32 vy
float32 ro
float32 phi
float32 ro_dot
